torch/csrc/jit_decomp_interface.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <ATen/core/Tensor.h>
+#include <ATen/core/function_schema.h>
+#include <c10/macros/Export.h>
+
+namespace torch {
+namespace autograd {
+namespace impl {
+
+struct TORCH_API JitDecompInterface {
```

Comment:
Should add some comments or docs about why this is necessary and what it is used for